packages/grafana-ui/src/components/PanelChrome/PanelStatus.tsx

```diff
@@ -0,0 +1,45 @@
+import { css } from '@emotion/css';
+import React from 'react';
+
+import { GrafanaTheme2, LoadingState } from '@grafana/data';
+
+import { useStyles2 } from '../../themes';
+import { ToolbarButton } from '../ToolbarButton/ToolbarButton';
+
+export interface Props {
+  state?: LoadingState;
```

Comment:
Should we call this prop `loadingState`, to have consistency with the PanelChrome prop?

Reply:
Update: https://github.com/grafana/grafana/pull/60147/commits/21cc1b965a6309185f20436b40ea8708df4cc508
I think this approach is cleaner 😊
Thank you 🙏